src/gui/modals/SetupDialog.cpp

```diff
@@ -1240,6 +1252,22 @@ void SetupDialog::resetBufferSize()
 	setBufferSize(DEFAULT_BUFFER_SIZE / BUFFERSIZE_RESOLUTION);
 }
 
+void SetupDialog::updateSampleRates()
+{
+	m_sampleRate->clear();
+	if (m_audioInterfaces->currentText() == "JACK (JACK Audio Connection Kit)")
+	{
+		m_sampleRate->setDisabled(true);
+	}
```

Comment:
Why do we disable it for JACK?

Reply:
Unlike with the other options, Jack clients all run synchronised to the jack master clock, so we can't change it in that case.